Makefile

```diff
@@ -0,0 +1,116 @@
+ifdef DOCKER_COMMON_ENV_PATH
+include $(DOCKER_COMMON_ENV_PATH)
+DOCKER_COMMON_ENV_PATH_FROM_PYTHON_DOCKER=$(DOCKER_CTX_FROM_PYTHON_DOCKER)/$(DOCKER_COMMON_ENV_PATH)
+endif
+
+ifdef DOCKER_SPECIFIC_ENV_PATH
+include $(DOCKER_SPECIFIC_ENV_PATH)
+DOCKER_SPECIFIC_ENV_PATH_FROM_PYTHON_DOCKER=$(DOCKER_CTX_FROM_PYTHON_DOCKER)/$(DOCKER_SPECIFIC_ENV_PATH)
+endif
+
+export
+
+NAMESPACE ?= local
+PROJECT_NAME ?= balancer_backend
+PYTHON_VERSION ?= 3.10
+
+DOCKER_APP_DEST ?= /app
+DOCKER_CTX_FROM_COMPOSE ?= ../../..
+DOCKER_CTX_FROM_PROJECT_ROOT ?= .
+DOCKER_CTX_FROM_PYTHON_DOCKER ?= ..
+DOCKER_ENTRYPOINT_DEST ?= /entrypoint.sh
+DOCKER_LOCAL_CMD ?= /bin/bash
+DOCKER_LOG_FOLDER_PATH ?= /root/.$(PROJECT_NAME)/logs
+DOCKER_NO_CACHE ?=  # Define with any value to disable caching on build
+DOCKER_PROJECT_ROOT_FROM_CTX ?= .
+DOCKER_PROJECT_SERVICE_NAME ?= balancer-backend
+DOCKER_PYTHON_DOCKER_FROM_CTX ?= Python-Docker
+DOCKER_REGISTRY ?= docker.fakecodeforphillydomain.com
```

Comment:
Is there a Code for Philly container registry we can use so that people don't need to build? We may want base env files as well.